README.md

```diff
@@ -20,14 +20,54 @@ npm install --save yieldbird_gpt_components
 ```tsx
 import React, { Component } from 'react'
 
-import MyComponent from 'yieldbird_gpt_components'
-import 'yieldbird_gpt_components/dist/index.css'
-
-class Example extends Component {
-  render() {
-    return <MyComponent />
-  }
-}
+import { AdManagerProvider, AdManagerSlot } from 'yieldbird_gpt_components'
+
+return (
+  <AdManagerProvider uuid='YOUR_YIELDBIRD_WRAPPER_UUID'>
+    <AdManagerSlot
+      adUnitPath='AD_UNIT_PATH/FOO'
+      size={[[120, 600], [160, 600]]}
+      optDiv='AD_UNIT_DIV_ID'
+    />
+  </AdManagerProvider>
+)
+```
+
+`AdManagerProvider` context component should be placed at the top of your React app. It is responsible for injecting GPT and Yieldbird Wrapper scripts, initializing variables and storing helper data.
+
+`AdManagerSlot` is a simple ad component, with properties similar to GPT slot. It is responsible for rendering ad in specified place. You can use it with following properties:
+| name | type | required | description |
+| :---- |  :----:  |  :----:  | :---- |
+| `adUnitPath` | string | true | Google AdManager ad unit path |
+| `size` | number | true | base slot sizes |
+| `optDiv` | string | true | name of DIV ID of given adUnit |
+| `targeting` | object | false | extra targeting object which can be used to pass aditional key-values pairs to GAM |
+| `sizeMapping` | array | false | array representation of size mapping GPT command calls. Each array element consists of two more arrays, representing viewport size and mapping which correspnds to GPT setup. |
```

Comment:
To tylko taka luźna sugestia, ale tutaj dodałbym linka do tej dokumentacji od GPT, np.

```suggestion
| `sizeMapping` | array | false | array representation of size mapping GPT command calls. Each array element consists of two more arrays, representing viewport size and mapping which correspnds to [GPT setup](www.google.pl). |
```

tylko zamiast adresu googla podać rzeczywisty :) Wydaje mi się, że będzie to przystępniejsze dla osób korzystających

Reply:
+1